tests/prioritization.rs

```diff
@@ -68,7 +68,7 @@ fn single_stream_send_large_body() {
 
 #[test]
 fn multiple_streams_with_payload_greater_than_default_window() {
-    let _ = ::env_logger::init();
+    let _ = ::env_logger::try_init();
```

Comment:
This is unrelated to the PR but it can cause tests to fail and it didn't seem worth splitting it out.